molecule/multimachine/cleanup.yml

```
@@ -30,7 +30,7 @@
   hosts: localhost
   gather_facts: false
   vars:
-    broker_name: molecule-{{ lookup('env', 'BUILD_NUMBER') }}
+    broker_name: molecule-{{ lookup('env', 'BRANCH_NAME')[:42] }}-{{ lookup('env', 'BUILD_NUMBER') }}
```

```suggestion
    broker_name: molecule-{{ lookup('ansible.builtin.env', 'BRANCH_NAME')[:42] }}-{{ lookup('ansible.builtin.env', 'BUILD_NUMBER') }}
```
I am not 100% sure, but maybe after OPSEXP-1372 this should be using FQN too?